### PR TITLE
New security process

### PR DESCRIPTION
**Pull request by dan-mckean**
Draft PR to share with Sec team for input and feedback.

### CODE REVIEW

File: handbook/engineering/security/process.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  * [Process overview](#process-overview)
+  * [How to work with us](#how-to-work-with-us)
+  * [Other processes](#other-processes)
+
+## Principles
+
+1. [Goals](https://about.sourcegraph.com/company/goals/guidelines) are something we strive for, whilst tracking and communicating progress.
+2. A work item is a piece of work (e.g., writing code, hiring a new teammate) that makes progress toward achieving a goal.
+3. Releases may be made up of N workitems, that may impact Y goals. Whilst this is true, we communicate both internally and externally progress towards those goals.
+4. Security by its various nature has public work items ([main repo](https://github.com/sourcegraph/sourcegraph) and private workitems ([security repo](https://github.com/sourcegraph/security-issues/)). Over time workitems should move from the private repository to the public repository once they can be made public. The ideal goal state is the lack of a private security repository.
```
Comment (ElizabethStirling) [inline handbook/engineering/security/process.md:16]: Nit - Should we update this to reference Jira instead?

File: handbook/engineering/security/process.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+## Process Overview
+
+### Planning & Roadmap
+
+1. We plan iterations and features prior to their execution, in a team planning session.
+2. We set one or more goals for the iteration.
+3. We write RFCs and solicit feedback ideally, prior to the start of an iteration, but especially with forethought in mind.
+4. We hold weekly team syncs and [track them here](https://docs.google.com/document/d/1l-JyN-hol2G6YXNqPsJsIgN2z3aZEzOW4-Julu4xthI).
+5. We report on our status and progress weekly in [tracking issues](https://github.com/sourcegraph/sourcegraph/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Atracking+label%3Ateam%2Fsecurity), and radiate communication.
```
Comment (ElizabethStirling) [inline handbook/engineering/security/process.md:27]: Nit - Jira again

File: handbook/engineering/security/process.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+Work stays in ProductBoard until it qualifies from "we might do this" to "we are going to". At that point, things transition over into Jira for more detailed planning and execution.
+
+### Execution
+
+Jira is intended only for **planned** work.
+  * Big work 
+    * Stuff we're **going to do** makes it into here via ProductBoard, where it gets broken down, ordered, and prioritized against other commited stuff
+  * Small items that don't fit with the strategic roadmap, but that need to be done soon
+  * Bugs
```
Comment (ElizabethStirling) [inline handbook/engineering/security/process.md:44]: Nit - should we add "asks from internal teams?"

File: handbook/engineering/security/process.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+### Execution
+
+Jira is intended only for **planned** work.
+  * Big work 
+    * Stuff we're **going to do** makes it into here via ProductBoard, where it gets broken down, ordered, and prioritized against other commited stuff
+  * Small items that don't fit with the strategic roadmap, but that need to be done soon
+  * Bugs  
+
+## How to work with us
+
```
Comment (ElizabethStirling) [inline handbook/engineering/security/process.md:47]: We should add a link to the handbook page about security questionnaires.

https://about.sourcegraph.com/handbook/sales/salessecurity

File: handbook/engineering/security/process.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+## Other processes
+
+### Learning / retrospective
+
+After the each release, we hold a [retrospective](https://about.sourcegraph.com/retrospectives). We try to understand the degree to which we achieved the goals we communicated at the beginning of the iteration. We identify what went well and what our opportunities for improvement. We actively choose one of the things we've learned, and target its improvement.
+
+### Deploying infrastructure
+
+Security develops infrastructure in the [Auxilliary project](https://console.cloud.google.com/home/dashboard?project=sourcegraph-server&_ga=2.42742757.1539584256.1606825468-757838940.1606655220). We work with the [Distribution team](https://about.sourcegraph.com/handbook/engineering/distribution) to deploy and test in dogfood, and promote to production. We are responsible for documentation and operating systems, and [Distribution](https://about.sourcegraph.com/handbook/engineering/distribution) helps make infrastructure production-ready based on our guidelines. Security acts as an input to Distribution.
```
Comment (ElizabethStirling) [inline handbook/engineering/security/process.md:64]: Nit - Not sure that most of this is true anymore. We don't deal with the auxiliary project, and don't really do much with testing or documentation. Would consider just removing this section.

cc @andreeleuterio  - thoughts?

File: handbook/engineering/security/index.md [hunk 6]
```diff
@@ -6,6 +6,14 @@
 
 We think that security is an enabler for the business. Sourcegraph is committed to proactive security, and addressing vulnerabilities in a timely manner. We approach security with a can-do philosophy, and look to achieve product goals while maintaining a positive posture, and increasing our security stance over time.
 
+## Members
+
+- [Dan Mckean](../../../company/team/index.md#dan-mckean-he-him) (Product Manager)
+- FQ2 engineering manager hire, [Bill Creager](../../../company/team/index.md#bill-creager) acting manager until then {#security-eng}
+  - [Elizabeth Stirling](../../../company/team/index.md#elizabeth-stirling-she-her)
+  - [André Eleuterio](../../../company/team/index.md#andré-eleuterio-hehim)
+  - FQ4 security engineer
```
Comment (ElizabethStirling) [inline handbook/engineering/security/index.md:15]: Nit - IIRC, our hiring plans have changed. CC @bcreager  - what's the current state after the board meeting?

**Review by ElizabethStirling — COMMENTED** [review]
Looks good, with a few nits. Will approve once we're in a more finalized state

File: handbook/engineering/security/process.md (hunk 5 above)
Comment (andreeleuterio) [reply handbook/engineering/security/process.md:64]: Agree. We also don't need the `Learning / retrospective` section.

File: handbook/engineering/security/process.md (hunk 1 above)
Comment (dan-mckean) [reply handbook/engineering/security/process.md]: Updating in branch and re-raising PR.

File: handbook/engineering/security/process.md (hunk 2 above)
Comment (dan-mckean) [reply handbook/engineering/security/process.md]: Updating in branch and re-raising PR.

File: handbook/engineering/security/process.md (hunk 3 above)
Comment (dan-mckean) [reply handbook/engineering/security/process.md:44]: Updating in branch and re-raising PR.

File: handbook/engineering/security/process.md (hunk 4 above)
Comment (dan-mckean) [reply handbook/engineering/security/process.md:48]: Updating in branch and re-raising PR.

File: handbook/engineering/security/process.md (hunk 5 above)
Comment (dan-mckean) [reply handbook/engineering/security/process.md]: Updating in branch and re-raising PR.

**Pull request closed without merging**